pulsar-io/kinesis/src/main/java/org/apache/pulsar/io/kinesis/KinesisSinkConfig.java

```diff
@@ -172,4 +172,10 @@ public enum MessageFormat {
             help = "Custom AWS STS port to connect to"
     )
     private Integer awsStsPort;
+
+    @FieldDoc(
+            defaultValue = "true",
+            help = "Enable aggregation. With aggregation, multiple user records could be packed into a single\n"
+                    + " KinesisRecord. If disabled, each user record is sent in its own KinesisRecord.")
+    private boolean enableAggregation = true;
```

Comment:
Rename to `aggregationEnabled` so that this is consistent with the Kinesis Producer API's `aggregationEnabled` setting.